charts/atlantis/values.yaml

```diff
@@ -88,6 +88,9 @@ orgAllowlist: <replace-me>
 # To reference an already existing Secret object with AWS credentials
 # awsSecretName: 'mysecretwithawscreds'
 
+# option to share process namespace with atlantis container
+shareProcessNamespace: false
```

Comment:
Please document the new value in the README

Reply:
hi @nitrocode this is now documented in the README